srcpkgs/j-devel/template

```diff
@@ -0,0 +1 @@
+../j/template
```

Comment:
This should just link to the main packages directory, `ln -s j srcpkgs/j-devel`

Reply:
Hi @Duncaen, I interpreted your comment as `ln -s ./j j-devel` - but please correct me if I've misunderstood. Thanks for your help!

Reply:
We generally just use the package name, i.e. `ln -s j j-devel` if you do it form the srcpkgs directory.
This makes it easier to just read the source package without having to resolve or parse any paths.